### PR TITLE
Add support for Parsely analytics

### PR DESCRIPTION
**Pull request by msukmanowsky**
See #1500 for additional information. This PR adds vendor configuration support for Parse.ly analytics.

Outstanding issues are the additional configuration variables we're requesting and the open discussions on client-side sessionization and engaged time implementations.

### CODE REVIEW

**Comment by msukmanowsky** [comment]
I signed it corporation name is Parsely Incorporated.

**Comment by amontalenti** [comment]
I also signed the CLA as Parse.ly's signing authority on this. Thanks.

**Comment by rudygalfi** [comment]
Thanks for the PR! cc @avimehta

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -59,6 +64,17 @@ export const ANALYTICS_CONFIG = {
           'st=${socialActionTarget}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'parsely': {
+    'host': 'srv.pixel.parsely.com',
+    'method': 'GET',
```
Comment (rudygalfi) [inline extensions/amp-analytics/0.1/vendors.js]: Is this intended to set the `transport` value?

https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#transport

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -59,6 +64,17 @@ export const ANALYTICS_CONFIG = {
           'st=${socialActionTarget}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'parsely': {
+    'host': 'srv.pixel.parsely.com',
```
Comment (rudygalfi) [inline extensions/amp-analytics/0.1/vendors.js]: The config spec has changed. The hostname is now part of the request. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#requests

File: extensions/amp-analytics/0.1/vendors.js [hunk 3]
```diff
@@ -59,6 +64,17 @@ export const ANALYTICS_CONFIG = {
...
+    'method': 'GET',
+    'requests': {
+      'baseHit': '/plogger/?rand={{timestamp}}&idsite={{apikey}}&url={{ampdocUrl}}&urlref={{documentReferrer}}&screen={{screenWidth}}x{{screenHeight}}%7C{{screenAvailWidth}}x{{screenAvailHeight}}%7C{{colorDepth}}&title={{title}}&date={{timestamp}}&ampid={{clientId}}'
+      'pageview': '{{baseHit}}&action=pageview',
```
Comment (jmadler) [inline extensions/amp-analytics/0.1/vendors.js]: add comma to baseHit, remove comma from pageview

Comment (cramforce) [inline extensions/amp-analytics/0.1/vendors.js]: The first comma is, indeed, crucial. We do, however, prefer the trailing comma in literals. Need to check the linter to actually enforce that.

File: extensions/amp-analytics/0.1/vendors.js [hunk 4]
```diff
@@ -40,6 +40,11 @@ export const ANALYTICS_CONFIG = {
       'scrollHeight': 'SCROLL_HEIGHT',
       'screenWidth': 'SCREEN_WIDTH',
       'screenHeight': 'SCREEN_HEIGHT'
+      // TODO: Requested by Parse.ly
+      //   screenAvailWidth (screen.availWidth)
+      //   screenAvailHeight (screen.availHeight)
+      //   colorDepth (screen.colorDepth)
```
Comment (rudygalfi) [inline extensions/amp-analytics/0.1/vendors.js]: The above variables captured by this issue: https://github.com/ampproject/amphtml/issues/1503

**Comment by msukmanowsky** [comment]
Apologies folks, the PR got a little muddled, will close and reopen.

**Pull request closed without merging**